R/build.r

```diff
@@ -155,12 +157,12 @@ build_demos <- function(pkg = ".") {
   demos <- readLines(file.path(demo_dir, "00Index"))
 
   pieces <- str_split_fixed(demos, "\\s+", 2)
-  in_path <- str_c(pieces[, 1], ".r")
+  in_path <- str_c(pieces[, 1], ".[rR]")
   filename <- str_c("demo-", pieces[,1], ".html")
   title <- pieces[, 2]
 
   for(i in seq_along(title)) {
-    demo_code <- readLines(file.path(demo_dir, in_path[i]))
+    demo_code <- readLines(Sys.glob(file.path(demo_dir, in_path[i])))
```

Comment:
Because now `in_path[i]` contains filenames with `".[rR]"` appended instead of `".r"` (see line [160](https://github.com/hadley/staticdocs/pull/99/files/d970b71e2d1aeaa7d4efb8fdef1b3ddac7bb0816#diff-f139653799d6fad71cdf6b25d156db3dR160)).